Upstream's justfile installs systemd sysusers.d/tmpfiles.d fragments and
a Debian-specific "*-start" wrapper script, none of which apply on a
runit system (Void handles system accounts via the template's
system_accounts/make_dirs, and greetd is launched directly per
02_fix_username_greetd.patch). Strip the systemd-only install targets
so `just install` doesn't try to ship files this package never builds.

--- a/justfile	2026-01-01 22:57:42.563588367 +0100
+++ b/justfile	2026-01-02 03:21:53.029096690 +0100
@@ -15,19 +15,9 @@
 # Systemd sysusers/tmpfiles components directories
 lib-dir := base-dir / 'lib'
 
-# sysusers.d
-sysusers-src := 'debian' / name + '.sysusers'
-sysusers-dst := lib-dir / 'sysusers.d' / name + '.conf'
-# tmpfiles.d
-tmpfiles-src := 'debian' / name + '.tmpfiles'
-tmpfiles-dst := lib-dir / 'tmpfiles.d' / name + '.conf'
-
 daemon-src := cargo-target-dir / 'release' / name + '-daemon'
 daemon-dst := base-dir / 'bin' / name + '-daemon'
 
-start-src := name + '-start.sh'
-start-dst := base-dir / 'bin' / name + '-start'
-
 dbus-src := 'dbus' / APPID + '.conf'
 dbus-dst := base-dir / 'share' / 'dbus-1' / 'system.d' / APPID + '.conf'
 
@@ -69,21 +59,14 @@
 
 # Install only debian package required files
 # The sysusers and tmpfiles files are automatically added
-install-debian:
+install:
     install -Dm0755 {{bin-src}} {{bin-dst}}
-    install -Dm0755 {{start-src}} {{start-dst}}
     install -Dm0755 {{daemon-src}} {{daemon-dst}}
     install -Dm0755 {{dbus-src}} {{dbus-dst}}
 
-# Installs files
-install: install-debian
-    install -Dm0644 {{sysusers-src}} {{sysusers-dst}}
-    install -Dm0644 {{tmpfiles-src}} {{tmpfiles-dst}}
-
 # Uninstalls installed files
 uninstall:
-    rm {{start-dst}} {{bin-dst}} {{daemon-dst}} {{dbus-dst}} {{sysusers-dst}} {{tmpfiles-dst}}
-
+    rm {{bin-dst}} {{daemon-dst}} {{dbus-dst}}
 # Vendor dependencies locally
 vendor:
     #!/usr/bin/env sh
